### ISSUE TITLE
Run npm build-compare script breaks html report

### ISSUE THREAD

**Issue by 4hardfire** [garris/BackstopJS#1154]
I was trying to compile the React App (that is responsible for the html report) using the npm script **build-compare**. Although, the generated index_bundle.js is not the same according to the latest version of the file that is already located at **compare/output/index_bundle.js**.
If we compile the React App and use the new generated index_bundle.js, when visiting the report page the images are not being shown.
Also, when trying to compile for the first time the following error has been thrown:

```
ERROR in ./compare/src/components/ecosystems/Header.js
Module not found: Error: Can't resolve '../organisms/topbar' in '/home/bernardo/Desktop/backstopjs/compare/src/components/ecosystems'
 @ ./compare/src/components/ecosystems/Header.js 21:14-44
 @ ./compare/src/components/App.js
 @ ./compare/src/index.js
```
To fix this error I just had to fix the location of the Topbar component that is being imported in  Header.js, so instead of **import Topbar from '../organisms/topbar';** we have to use **import Topbar from '../organisms/Topbar';**

It would be nice if someone can help me understanding where the problem is coming with the compiled version of index_bundle.js since I will probably need to implement a pager for those cases  where we have thousands of scenarios.

**Comment by garris** [comment]
Hmm.  I think I was probably the last eng in there.  Didn't have any compile issues.  Are you on latest?  Also, are you on MacOS?

**Comment by garris** [comment]
Also,  happy to pull in improvements -- but please open an issue first where we can discuss/align on the changes first.  Thanks!

**Comment by 4hardfire** [comment]
I was using an outdated version of node and I think that was the problem. I was using Ubuntu though. Then I tested this on my Mac but in this case I didn't have node installed so I've used a docker image to run npm and it worked fine.
I've done the same in Ubuntu and now it works as expected so I think this can be closed :).

About the new improvements, I've already forked the repository and have a pager showing on the report. The only thing I wanted to do now is to have the pager options in the config.js so I could load those in the React App. Anyway, it would be nice to integrate this in BackstopJS so I will definitely open an issue so we can discuss/align on the changes like you said.

And thanks for the support :)
